sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/Status.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Authorization.Models
+{
+    /// <summary> The status of the role assignment schedule. </summary>
+    public readonly partial struct Status : IEquatable<Status>
```

Comment:
Single word class name, this one is too generic, can we rename it to a more specific name?